var/spack/repos/builtin/packages/madis/package.py

```diff
@@ -32,7 +32,7 @@ class Madis(MakefilePackage):
 
     def setup_build_environment(self, env):
         fflags = []
-        if self.spec.satisfies("%gcc@10:"):
+        if self.spec.satisfies("%gcc@10:") or self.spec.satisfies("%apple-clang"):
```

Comment:
This works, but better would be to detect if the Fortran compiler is gfortran 10+ - see ESMF package.

Reply:
Done, and builds successfully in https://github.com/JCSDA/spack-stack/actions/runs/6003773051/job/16282947966?pr=743